### ISSUE TITLE
Confirm 404 pages won't link to pending or draft posts

### ISSUE THREAD

**Issue by dreeves** [beeminder/blog#305]
Just making sure...

**Comment by narthur** [comment]
Yes, that's correct. [The 404 page](https://github.com/beeminder/blog/blob/master/src/pages/404.astro) uses [the getPosts() function](https://github.com/beeminder/blog/blob/master/src/lib/getPosts.ts) to retrieve all posts which may be listed, and does not pass the function any options. If no options are passed to `getPosts()`, its default behavior is to [only include published posts](https://github.com/beeminder/blog/blob/master/src/lib/getPosts.ts#L6). Thus, the 404 page only displays published posts.

Of course, a manual check to double-check that's working as expected wouldn't hurt.
